File: udemy/java/vendas/src/vendas/Vendas.java
package vendas;
import javax.swing.JOptionPane;
public class Vendas {
	public static void main(String[] args) {
		float total, vendas, fixo;
		vendas = Float.parseFloat(JOptionPane.showInputDialog("informe o numero de vendas"));
		vendas = (vendas*15)/100;
		fixo =  800;
		total = fixo + vendas;
		JOptionPane.showMessageDialog(null,"O valor total eh: "+total);
	}

}

File: udemy/pascal/projetos/vendas.pas
program vendas;
var comissao, fixo, total, vendas:real;
begin write ('informe o valor das vendas: ');
readln (vendas);
comissao:= vendas * (15 / 100);
fixo:= 800;
total:= fixo + comissao;
write ('O valor qque recebera eh: ', total: 6:2);
readln ();
end.
